secrets/secrets-vault/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "secrets-vault"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = { version = "1.0.86", default-features = false, features = ["std"] }
+# TODO: re-enable upstream async-nats once it supports the feature set needed by wRPC.
+# async-nats = { version = "0.35", default-features = false, features = ["ring", "server_2_10"] }
+async-nats = { package = "async-nats-wrpc", version = "0.35.1", features = ["ring", "server_2_10"] }
+axum = { version = "0.7.5", default-features = false, features = ["http1", "json", "tokio", "tracing"] }
+bytes = { version = "1", default-features = false }
+clap = { version = "4.5.4", features = ["derive", "env", "string"] }
+data-encoding = { version = "2.6.0" }
+ed25519-dalek = { version = "2.1.1", features = ["alloc", "pkcs8"] }
+futures = { version = "0.3.30", default-features = false, features = [] }
+jsonwebtoken = { version = "9.3.0" }
+nkeys = { version = "0.4.2", features = ["xkeys"] }
+serde = { version = "1.0.203", default-features = false, features = ["std"] }
+serde_json = { version = "1.0.117", default-features = false, features = ["std"] }
+sha2 = { version = "0.10.8" }
+tokio = { version = "1.38.0", default-features = false, features = ["full"] }
+tracing = { version = "0.1.40", default-features = false, features = [] }
+tracing-subscriber = { version = "0.3.18", default-features = false, features = ["fmt", "env-filter"] }
+vaultrs = { version = "0.7.2", default-features = false, features = ["rustls"] }
+# TODO: update this to use the published crate once it exists.
+wasmcloud-secrets-types = { version = "0.1.0", git = "https://github.com/wasmcloud/wasmCloud.git", branch = "main" }
```

Comment:
Ready now!

Reply:
I will also need [`secrets-client` published](https://github.com/wasmCloud/wasmCloud/pull/2592) so that there isn't a conflict between the types used by the git-based `secrets-client` and the published `secrets-type` crate.

Reply:
Tag pushed 😉 Should be ready here in just a bit